.github/workflows/pr_test_cpu.yml

```diff
@@ -18,28 +18,26 @@ jobs:
       fail-fast: false # TODO: enable it when CI be stable
       matrix:
         os: ['Ubuntu-latest', 'Windows-latest', 'MacOS-latest']
-        pytorch-dtype: ['float16', 'float32', 'float64']
+        pytorch-dtype: ['float32', 'float64']
```

Comment:
maybe, an option here is to test fp32 on all OS, but the fp64 just on ubuntu

Reply:
+1min a separated pr